### ISSUE TITLE
x:Uid localization doesn't work if the property isn't defined in XAML

### ISSUE THREAD

**Issue by MatFillion** [unoplatform/uno#921]
## I'm submitting a...
 - Bug report (I searched for similar issues and did not find one) 

## Current behavior
```
<TextBlock x:Uid="AboutFlyoutEULA"
      FontSize="{ThemeResource BodyFontSize}"
      TextWrapping="Wrap" />
```
Will display no text even though a resource with key AboutFlyoutEULA.Text exists in the .resw file

Additional info:
```
<TextBlock x:Uid="AboutFlyoutEULA"
      Text="some text"
      FontSize="{ThemeResource BodyFontSize}"
      TextWrapping="Wrap" />
```
Adding a text property as seen above will display the proper value of AboutFlyoutEULA.Text defined in the resw. (instead of "some text").

## Expected behavior
```
<TextBlock x:Uid="AboutFlyoutEULA"
      FontSize="{ThemeResource BodyFontSize}"
      TextWrapping="Wrap" />
```
Should display the value of resource defined by key AboutFlyoutEULA.Text in the .resw file

## Minimal reproduction of the problem with instructions
See above code snippets

## Environment

```
Nuget Package:

Package Version(s):

Affected platform(s):
- [ ] iOS (not tested)
- [x] Android
- [x] WebAssembly
- [ ] Windows
- [ ] Build tasks

Visual Studio
- [x] 2017 (version: )
- [x] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins
- [ ] Resharper (version: )

**Comment by MatFillion** [comment]
@MakanWG

**Comment by MakanWG** [comment]
Seems like it is not always the case. Anyway there is a repro here https://github.com/nventive/calculator on the Uno branch.
- Clone the repo
- Launch on either Android or wasm
- Open the menu and click on About at the very bottom
- Notice the informations are well displayed

Then open AboutFlyout.xaml and remove the text property in the text of one of the Hyperlinks.

- Follow the same steps as above.
- Notice this link is no longer displayed

**Comment by davidjohnoliver** [comment]
This is partially fixed by #936 , but still not working correctly for attached properties.

**Comment by darenm** [comment]
@davidjohnoliver I noticed that this issue applies to **ContentDialog** properties such as **Title**, **PrimaryButtonText** and **SecondaryButtonText** in WASM.

If the **ContentDialog** is defined as below, the resources are not applied to **Title**, **PrimaryButtonText** and **SecondaryButtonText**:

```xml
<ContentDialog
    x:Class="TimeEntryUno.Shared.Views.Login.LoginForm"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:TimeEntryUno.Shared.Views.Login"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d"
    x:Uid="LoginDialog"
    PrimaryButtonClick="ContentDialog_PrimaryButtonClick"
    SecondaryButtonClick="ContentDialog_SecondaryButtonClick">
```

However, if the properties are given default values in XAML, as below, it renders correctly 

```xml
<ContentDialog
    x:Class="TimeEntryUno.Shared.Views.Login.LoginForm"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:TimeEntryUno.Shared.Views.Login"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d"
    x:Uid="LoginDialog"
    Title="TITLE"
    PrimaryButtonText="Button1"
    SecondaryButtonText="Button2"
    PrimaryButtonClick="ContentDialog_PrimaryButtonClick"
    SecondaryButtonClick="ContentDialog_SecondaryButtonClick">
```
